modules/RTC/JitsiLocalTrack.js

```diff
@@ -88,21 +86,13 @@ export default class JitsiLocalTrack extends JitsiTrack {
         this.sourceId = sourceId;
         this.sourceType = sourceType;
 
-        if (browser.usesNewGumFlow()) {
-            // Get the resolution from the track itself because it cannot be
-            // certain which resolution webrtc has fallen back to using.
-            this.resolution = track.getSettings().height;
-
-            // Cache the constraints of the track in case of any this track
-            // model needs to call getUserMedia again, such as when unmuting.
-            this._constraints = track.getConstraints();
-        } else {
-            // FIXME Currently, Firefox is ignoring our constraints about
-            // resolutions so we do not store it, to avoid wrong reporting of
-            // local track resolution.
-            this.resolution = browser.isFirefox() ? null : resolution;
-        }
+        // Get the resolution from the track itself because it cannot be
+        // certain which resolution webrtc has fallen back to using.
+        this.resolution = track.getSettings().height;
```

Comment:
we don't have this on mobile, it throws an error because it's not implemented

Reply:
For now on RN I went back to using the resolution passed to the constructor of JitsiLocalTrack and I am passing `{ video: true }` as constraints when unmuting video.

Reply:
That's a step backwards from the previous behavior, the resolution was respeccted. Let me look into implementing those 2 to see if we can avoid leaving this for later.

Reply:
Ok sure, will wait for you to implement it. We have another option as well - we could construct the constraints from the options and then use those constraints while unmuting video (just saying).